soccer/gameplay/tactics/stopped/circle_near_ball.py

```diff
@@ -155,11 +139,27 @@ def execute_running(self):
         for b in self.all_subbehaviors():
             if b.robot is not None:
                 num_robots += 1
+        #if the number of robots has changed, recreate move behaviors to match new number of robots
+        if (self.num_robots != num_robots):
+            self.num_robots = num_robots
+            self.remove_all_subbehaviors()
+            i = 0
+            for pt in range(6):
```

Comment:
Do you know if this `6` can be replaced with a constant? If not, we should open a new issue to refactor all these 6s away into references to Constants.hpp